cobra/util/optlang.py

```diff
@@ -0,0 +1,46 @@
+def split_bounds(lower_bound, upper_bound):
```

Comment:
The separation is good, though nothing wrong with extra 'floating' function definitions in a Reaction module (although correction that module name to reaction will reduce the feeling that it's only for the Reaction class, for a different cleanup PR)